packages/@okta/vuepress-site/docs/guides/sign-into-web-app/get-user-info/go/getuserinfo.md

```diff
@@ -0,0 +1,29 @@
+After setting up the previous steps, we stored the profile data in the `sessionStore`:
```

Comment:
"we stored" -> "you store"
Or maybe it's easier to use "we" everywhere instead of "you"?